Integrations/integration-Lastline.yml

```diff
@@ -0,0 +1,293 @@
+description: Submitting resources for analysis and obtaining the results
+configuration:
+- display: Server URL (e.g. https://user.lastline.com)
+  name: url
+  defaultvalue: http://user.lastline.com
+  type: 0
+  required: true
+- display: API Key for accessing Lastline APIs
+  name: api_key
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Token for accessing Lastline APIs
+  name: api_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "True"
+  type: 8
+  required: false
+- display: Score threshold for marking file or URL as malicious
+  name: malicious_score
+  defaultvalue: "50"
+  type: 0
+  required: false
+- display: Score threshold to present analysis in war room
+  name: full_report_score
+  defaultvalue: "50"
+  type: 0
+  required: false
+script:
+  script: |+
+    var lastlineDefaults = {interval:10, timeout:60};
+
+    var serverUrl = params.url;
+    if (serverUrl[serverUrl.length - 1] !== '/') {
+        serverUrl += '/';
+    }
+
+    var doReq = function(method, path, parameters, body) {
+        if (!parameters) {
+            parameters = {};
+        }
+        parameters.key = params.api_key;
+        parameters.api_token=params.api_token;
+        parameters.full_report_score=params.full_report_score;
+
+        console.log(serverUrl + path + encodeToURLQuery(parameters));
+        var result = http(
+            serverUrl + path + encodeToURLQuery(parameters),
+            {
+                Headers: {'Content-Type': ['application/json']},
+                Method: method,
+                Body: ''
+            }
+        );
+        console.log(JSON.stringify(result));
+        if (result.StatusCode < 200 && result.StatusCode > 299) {
+            throw 'Failed to perform request ' + path + ', request status code: ' + result.StatusCode;
+        }
+        var obj = JSON.parse(result.Body);
```

Comment:
I started wrapping the parse in try catch just in case to display nice error - see VT, XFE, others